src/sentry/models/grouphash.py

```diff
@@ -26,9 +26,6 @@ class State:
         UNLOCKED = None
         LOCKED_IN_MIGRATION = 1
 
-        # This hierarchical grouphash should be ignored/skipped for finding the group.
-        SPLIT = 2
```

Comment:
This got added in https://github.com/getsentry/sentry/pull/25481

Reply:
Are there any rows still using this value? I've usually seen the constant kept around with a deprecated comment for compatibilitiy.

Reply:
This query returns that 1133 rows are in the LOCKED_IN_MIGRATION and none on SPLIT.
```
select
  state,
  count(*)
from
  getsentry.sentry_grouphash as gh
where
  gh.state is not null
group by gh.state;
```

Reply:
thanks for checking -- if this isn't incompatible for self hosted, then lgtm!